packages/@guardian/source-react-components-development-kitchen/src/expanding-wrapper/ExpandingWrapper.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import { css } from '@emotion/react';
+import { neutral } from '@guardian/source-foundations';
+import { useState } from 'react';
+import type { ReactElement } from 'react';
+import { ExpandingWrapper } from './ExpandingWrapper';
+
+/**
+ * [Storybook](https://guardian.github.io/source/?path=/docs/packages-source-react-components-label--playground) •
+ * [Design System](https://theguardian.design/2a1e5182b/p/40151e-label/b/86af7d) •
+ * [GitHub](https://github.com/guardian/source/tree/main/packages/@guardian/source-react-components/src/label/Label.tsx) •
+ * [NPM](https://www.npmjs.com/package/@guardian/source-react-components)
+ *
+ * An Expanding Wrapper surrounds a bit of content that can be collapsed and expanded
+ * Note: When collapsed, there should be no focusable elements within the wrapper.
+ *
+ * The following themes are supported: `light`.
+ * */
```

Comment:
is this comment correct for this story? the links refer to `source-react-components/src/label`?

Reply:
Right you are - have updated